lib/peach.ex

```diff
@@ -1,88 +1,98 @@
 defmodule Peach do
-  @moduledoc """
-  Documentation for `Peach`.
-  """
-
-  @doc """
-  Hello world.
-
-  ## Examples
-
-      iex> Peach.hello()
-      :world
-
-  """
-  def hello do
-    :world
-  end
-
   @doc """
   Normalize text
   Unicode NFKC (Normalisation Form Compatibility Composition) normalisation.
   """
-  def normalise_text do
-    # return normalize('NFKC', text)
-    :to_do
+  def normalise_text(phrase) do
+    # https://hexdocs.pm/elixir/1.9.4/String.html#normalize/2
+    :unicode.characters_to_nfc_binary(phrase)
```

Comment:
@nathanbegbie The reference implementation uses nfkc. k for compatible :-)
As far as I understand this is better for application like searching and text classification.
Is there a reason you used nfc?

Reply:
Gonna be honest, I was lazy and that's what the Elixir implementation said was available from the erlang library. But I see [here](http://erlang.org/doc/man/unicode.html#characters_to_nfkc_binary-1) that it is an option. Will make the change